.github/workflows/prerelease.yml

```diff
@@ -85,9 +85,8 @@ jobs:
           flavor: |
             latest=false
           tags: |
-            type=semver,pattern={{version}}+py${{ matrix.python-version }}
+            type=match,pattern=v(.*),group=1,suffix=-py${{ matrix.python-version }}
```

Comment:
This follows the `type=match` pattern which will strip the `v` off our tag: https://github.com/docker/metadata-action#typematch